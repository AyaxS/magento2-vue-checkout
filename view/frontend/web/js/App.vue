<template>
  <div class="checkout">
    <TheShippingStep/>
    <ThePaymentStep/>
    <TheSuccessStep/>
    <ProductsList/>
    <TotalsData/>
  </div>
</template>

<script>
import TheShippingStep from './components/steps/TheShippingStep.vue'
import ThePaymentStep from './components/steps/ThePaymentStep.vue'
import TheSuccessStep from './components/steps/TheSuccessStep.vue'
import TotalsData from './components/TotalsData.vue'
import ProductsList from './components/ProductsList.vue'

export default {
  components: {
    TheShippingStep,
    ThePaymentStep,
    TheSuccessStep,
    TotalsData,
    ProductsList
  }
}
</script>

<style src="vue-multiselect/dist/vue-multiselect.min.css"></style>
