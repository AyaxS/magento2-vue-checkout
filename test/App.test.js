import { shallowMount } from '@vue/test-utils'
import App from '../view/frontend/web/js/App.vue'

describe('App.test.js', () => {

  it('has the expected html structure', () => {
    const wrapper = shallowMount(App)
    expect(wrapper.element).toMatchSnapshot()
  })
})
