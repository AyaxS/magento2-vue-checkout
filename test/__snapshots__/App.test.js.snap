// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.test.js has the expected html structure 1`] = `
<div
  class="checkout"
>
  <theshippingstep-stub />
   
  <thepaymentstep-stub />
   
  <thesuccessstep-stub />
   
  <productslist-stub />
   
  <totalsdata-stub />
</div>
`;
